### PR TITLE
Defudding of the fudd round

### PR DESCRIPTION
**Pull request by Holli-Git**
#### Summary
Bugfixes "Description fix"

#### Purpose of change

Stopping power is a stupid phrase and I hate it, so I've updated it to be less fuddy.

#### Describe the solution

Changes the description of the round.

#### Describe alternatives you've considered

Remove any and all ammo, returning to the time of true stopping power that is of trebuchets. 

#### Testing

Do I have to test it if it's a description change? Serious question.

### CODE REVIEW

**Comment by Scorch-Mechanic** [comment]
> "It has good power but above average recoil compared to ~~it's~~ its now more widespread counterpart, 9mm."

"its" is the possessive, not "it's"

**Review by Maleclypse — APPROVED** [review]

**Comment by Alex-Folts** [comment]
> Stopping power is a stupid phrase

Yet it is [valid term](https://en.wikipedia.org/wiki/Stopping_power), blame english for didn't coin better sounding name for a thing. Not the end of the world if you change description though.

**Comment by Paprika-H** [comment]
> > "It has good power but above average recoil compared to ~it's~ its now more widespread counterpart, 9mm."
> 
> "its" is the possessive, not "it's"

Ah, thanks for catching that! "Its" now fixed

**Review by kevingranade — CHANGES_REQUESTED** [review]
Replacing "stopping power" with "power" is no good, the first is a term of art regardless of misuse that occurs, but the later is not a typical use at all.
If you want to work around this you need to rewrite the description more thoroughly.

**Comment by Paprika-H** [comment]
> Replacing "stopping power" with "power" is no good, the first is a term of art regardless of misuse that occurs, but the later is not a typical use at all. If you want to work around this you need to rewrite the description more thoroughly.

I really need to think these things more thoroughly, my apologies. I've adjusted it now

**Comment by PatrikLundell** [comment]
If you've made a change to a description, I'd say you'd need to verify that the changed description actually appears in the game (you could potentially have made the mistake of editing a copy of the file, or the object you edited may be a duplicate the loading of the game ignores [unless duplicate definitions are now flagged], etc.).
Whether you'd need to verify every changed instance when you've made a bunch of them is a judgement call, but if it's a matter of debug spawning items and looking at their descriptions it should be easy to do, and there would be little reason not to verify it in that case (but posting images of each and every item before and after is overkill, while one example item provides an illustration of the change within its context).

File: data/json/items/ammo/45.json [hunk 1]
```diff
@@ -4,14 +4,14 @@
     "copy-from": "45_jhp",
     "type": "AMMO",
     "name": { "str_sp": ".45 ACP FMJ" },
-    "description": ".45 ACP ammunition with 230gr FMJ bullets.  Noted for its stopping power, the .45 ACP round has been common for over a century.",
+    "description": ".45 ACP ammunition with 230gr FMJ bullets.  Due to the projectile weight causing significant tissue damage, and previous adoption by the US military, the .45 ACP round has been common for over a century.",
```
Comment (kevingranade) [inline data/json/items/ammo/45.json:7]:
```suggestion
    "description": ".45 ACP ammunition with 230gr FMJ bullets.  Due to a reputation for effectiveness, the .45 ACP round has been common for over a century.",
```

**Pull request merged**